index.html

```diff
@@ -379,8 +379,13 @@ <h2 id="hidden-display-surfaces">
           A <a>display surface</a> that is being shared may temporarily or permanently become
           inaccessible to the application because of actions taken by the operating system or user
           agent. What makes a <a>display surface</a> considered inaccesible is outside the scope
-          of this specification, but examples MAY include a <a>monitor</a> disconnecting or an
-          <a>application</a>, <a>window</a> or <a>browser</a> closing or becoming minimized.
+          of this specification, but examples MAY include a <a>monitor</a> disconnecting, an
+          <a>application</a>, <a>window</a> or <a>browser</a> closing or becoming minimized,
```

Comment:
Drive-by: As far as I can tell (from [this test](https://jsfiddle.net/k8wLn7uv/17/)), Firefox does not mute minimized windows (at least on Mac). Safari does not yet support windows/tabs, so I am unsure what their intentions there are. As long as we don't actually have a UA that either does so or intends to do so, can we get rid of this example?

Drive-by^2: Btw, @jan-ivar, I see the odd pattern below when minimizing a window that's being captured. It seems like the minimization animation on Mac is being captured by Firefox, then gets stuck on the last frame. Any idea if this is a known issue?
![Screen Shot 2022-01-31 at 14 24 48](https://user-images.githubusercontent.com/22117736/151803137-454e5e3c-5d6b-4cf3-b1fb-d3d6f081db43.png)